### ISSUE TITLE
Compiling error : widgets

### ISSUE THREAD

**Issue by LDAmorim** [AMReX-Codes/Amrvis#3]
Hi,

Sorry to bother you. I tried to compile Amrvis according to the documentation in:
https://amrex-codes.github.io/amrex/docs_html/Visualization.html#

But I get the error message:
```
Error: attempt to add non-widget child "DropSiteManager" to parent "amrvis2d.gnu.ex" which supports only widgets
```
when I try to use it.

Could you please help me understand what I am doing wrong?

Thanks,
Diana

**Comment by atmyers** [comment]
This in on OS X I assume?

**Comment by LDAmorim** [comment]
Hi,
Sorry for the late reply.
Yes, it is.

**Comment by atmyers** [comment]
I was able to install and run Amrvis on my Mac by using homebrew to install Openmotif, as described here: https://brewinstall.org/Install-openmotif-on-Mac-with-Brew/

Note that this is on OS X 10.11.

**Comment by LDAmorim** [comment]
Hi,
Sorry again for the delay in my reply.
I installed openmotif again, but as you suggested, this time using homebrew.
Then I re-compiled amrvis, again without a problem. But when I ran the executable, I encountered the same issue.

So I tried something I read online. I re-installed Xquartz with homebrew and ran the command:
```
export DYLD_LIBRARY_PATH=/opt/X11/lib/flat_namespace
```
(X11 is in /opt in my computer)
Now I can open the widget and the pot##### files ().

Thank you!
Diana
